### DISCUSSION TITLE
为啥我闺女在迁校之后就变得闷闷不乐了，当爸爸的该怎么办啊

### DISCUSSION THREAD

**Discussion by mystar32** [share121/inter-knot#5190] [绝区零]
前段时间我闺女的学校附近突然发现一个伴生空洞被迫迁校了。这里还有个之前拍的照片
![IMG_20240919_023446](https://github.com/user-attachments/assets/07c3bc82-f579-40c7-a200-a38e54ee7c91)
唉，看看我闺女之前，活泼开朗，学习成绩还好，还有两个好朋友一起玩，但是自从前段时间学校附近突然出现一个伴生空洞不得不迁校，我闺女整体闷闷不乐的。原来的玩伴也就剩一个了。唉，成绩再下降可怎么办啊。
说起来听人说他们三个当时进空洞了，你们说那个消失的孩子会不会是已经……我还听说这次空洞灾害是因为那个校长擅自开采以太资源导致的，结果他还升职了。
网友们有没有也在这所小学上学的孩子，或者知道点内幕的，要真是那校长让我家孩子变成这样的我非找他谈谈(｀へ´)=3

**Comment by bangboosama** [comment]
那个校长已经去吃公家饭了
